scripts/exrrfs_post.sh

```diff
@@ -376,6 +382,28 @@ fi
 
 if [ -f PRSLEV.GrbF${post_fhr} ]; then
   wgrib2 PRSLEV.GrbF${post_fhr} -set center 7 -grib ${bgdawp} >>$pgmout 2>>errfile
+  if [ $SUBH_GEN = 1 ]
+  then
+  bgdawp_subh_combo=${DATA}/${net4}.t${cyc}z.prslev.${gridspacing}.subh.f${fhr}.${gridname}.grib2
+  bgdawp_subh=${DATA}/PRSLEV.GrbF${fhr}.00
+  wgrib2 ${bgdawp} -not_if 'ave fcst' | grep -F -f ${FIX_UPP}/subh_fields.txt | wgrib2 -i -grib ${bgdawp_subh}  ${bgdawp}
+
+  fhrm1tmp="$((10#$fhr-1))"
+  fhrm1=`printf "%02d\n" $fhrm1tmp`
+  echo new fhrm1 is $fhrm1
+# expect this will need to be changed due to future umbrella directory changes
+  tm15=${DATA}/../rrfs_post_${envir}_${cyc}_f0${fhrm1}-45-00/PRSLEV.GrbF${fhrm1}.45
+  tm30=${DATA}/../rrfs_post_${envir}_${cyc}_f0${fhrm1}-30-00/PRSLEV.GrbF${fhrm1}.30
+  tm45=${DATA}/../rrfs_post_${envir}_${cyc}_f0${fhrm1}-15-00/PRSLEV.GrbF${fhrm1}.15
```

Comment:
Yes this directory structure will likely need to be updated but this is fine for now.